natlas-server/app/admin/routes.py

```diff
@@ -26,11 +25,11 @@
     ScopeLog,
     UserInvitation,
 )
-from app.auth.wrappers import is_authenticated, is_admin
+from app.auth.wrappers import is_admin
 
 
 @bp.route("/", methods=["GET", "POST"])
-@is_authenticated
+@login_required
```

Comment:
Wish I could take credit for it. `is_authenticated` is used all over the place and has a check for whether or not the current app is configured for LOGIN_REQUIRED or not. `login_required` is supplied by flask-login and doesn't have that check, it just flat out refuses any anonymous users (which is definitely what we want for admin routes and user routes, since neither of them should ever be accessible to anonymous users)